README.md

```diff
@@ -32,6 +32,12 @@
 | 1    | Random Forest | 1024-bit ECFP4 | 1.3178 +- 0.0081 | 0.6716 +- 0.0059 | [Mufei Li](mufeili1996@gmail.com) | [Paper](https://www.stat.berkeley.edu/~breiman/randomforest2001.pdf), [Code](./examples) | Dec 26th, 2020 |
 | 2    | GCN           | GraphConv      | 1.6450 +- 0.1325 | 0.5244 +- 0.0200 | [Mufei Li](mufeili1996@gmail.com) | [Paper](https://arxiv.org/abs/1609.02907), [Code](./examples)                            | Dec 26th, 2020 |
 
+### PDBbind (core)
+
+| Rank | Model         | Featurization  | Test RMSE     | Validation RMSE | Contact                           | References                                                                               | Date           |
```

Comment:
Could we also add R^2 here? I think RMS is more physically relevant, but R^2 is what we've used in the original ACNN paper